Source/NSProgress.m

```diff
@@ -284,9 +308,16 @@ - (BOOL) isIndeterminate
   return internal->_indeterminate;
 }
 
+- (BOOL) indeterminate
+{
+  return internal->_indeterminate;
+}
+
 - (void) setIndeterminate: (BOOL)flag
 {
+  [self willChangeValueForKey: @"indeterminate"];
```

Comment:
I think this is unnecessary as the basic KVO mechanism should create this automatically. This is also true for some of the following methods. We should only need to add willChange.. and didChange.. methods to support different names and even for these there was a better way, if I remember correctly.